vector/src/main/java/im/vector/app/features/rageshake/RageShake.kt

```diff
@@ -46,7 +46,7 @@ class RageShake @Inject constructor(private val activity: FragmentActivity,
 
         shakeDetector = ShakeDetector(this).apply {
             setSensitivity(vectorPreferences.getRageshakeSensitivity())
-            start(sensorManager)
+            start(sensorManager, SensorManager.SENSOR_DELAY_GAME)
```

Comment:
I don't think we'll have any issues but wanted to highlight this means we've changed from`SENSOR_DELAY_FASTEST` to `SENSOR_DELAY_GAME`

I wonder if `SENSOR_DELAY_UI` could be more suited? haven't tested, assuming from the docs...

```
/** get sensor data as fast as possible */
public static final int SENSOR_DELAY_FASTEST = 0;
/** rate suitable for games */
public static final int SENSOR_DELAY_GAME = 1;
/** rate suitable for the user interface  */
public static final int SENSOR_DELAY_UI = **2;**
```

Reply:
TBH I haven't checked SENSOR_DELAY_UI. The change was necessary because when using SENSOR_DELAY_FASTEST we need to add a permission in the manifest.
Rageshake still works with SENSOR_DELAY_GAME, I did not investigate more.